docs/source/run/service.rst

```diff
@@ -0,0 +1,121 @@
+  (:attr:`~sopel.config.core_section.CoreSection.homedir`)
+
+.. important::
+
+    Do **not** run Sopel as the **root** user. And avoid if possible to run
+    with a privileged user.
+
+
+Running with systemd
+====================
+
+`systemd`__ is a service manager used by Ubuntu and most Debian-derived
+Linux distributions. Here is an example of a service unit file to run Sopel as
+a service, placed at ``/etc/systemd/system/sopel.service``:
+
+.. code-block:: systemd
+
+    [Unit]
+    Description=Sopel IRC bot
+    Documentation=https://sopel.chat/
+    After=network-online.target
+
+    [Service]
+    Type=simple
+    User=sopel
+    RuntimeDirectory=sopel
+    StateDirectory=sopel
+    LogsDirectory=sopel
+    PIDFile=/run/sopel/sopel.pid
+    ExecStart=/home/sopel/.local/bin/sopel start
+    Restart=on-failure
+    RestartPreventExitStatus=2
+    RestartSec=30
+    Environment=LC_ALL=en_US.UTF-8 SOPEL_CONFIG_DIR=/etc/sopel
+
+    [Install]
+    WantedBy=multi-user.target
+
+.. __: https://systemd.io/
+
+For this service to work you will need the following:
+
+* to create a ``sopel`` user
+* to install ``sopel`` with that user (or to change the path in the
+  ``ExecStart``)
+* to place your Sopel configuration file into ``/etc/sopel/default.cfg``, and
+  to setup its access rights so that the ``sopel`` user can read and write it
+* and to edit the ``[core]`` section of this configuration file to set the
+  appropriate paths:
+
+  .. code-block:: ini
+
+      [core]
+      homedir = /var/lib/sopel
+      pid_dir = /run/sopel
+      logdir = /var/log/sopel
+
+These paths are required for Sopel to know where to put its files (data, cache,
+PID file, logs, etc.).
+
+And with that, you will be able to start/stop your bot with
+``systemctl start sopel`` and ``systemctl stop sopel``. Once you are sure it
+works, you can enable the service with ``systemctl enable sopel`` so that
+the service will start automatically at boot.
+
+.. seealso::
+
+    There is more to say about systemd, its service unit files, and the
+    ``systemctl`` command. Be sure to read `systemd.unit's documentation`__ for
+    more information.
+
+.. __: https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html
+
+
+Example of configuration for libera
+===================================
+
+To put everything together, let's say you want to run a bot for your channel
+on the `libera.chat network`__. For that, here is a configuration file you will
+need to put at ``/etc/sopel/default.cfg``:
+
+.. code-block:: ini
+
+    [core]
+    nick = <Your Bot Nick>
+    host = irc.libera.chat
+    port = 6697
+    use_ssl = yes
+    verify_ssl = yes
+    owner = <Your Nick>
+    channels =
+        "#yourchannel"
+    homedir = /var/lib/sopel
+    pid_dir = /run/sopel
+    logdir = /var/log/sopel
+
+Make sure to change ``<Your Bot Nick>`` by your bot's nick, as "Sopel" is
+already taken; and set your own's nick as the owner instead of ``<Your Nick>``.
```

Comment:
```suggestion
Make sure to replace ``<Your Bot Nick>`` with your bot's nick, as "Sopel" is
already taken; and set your own nick as the owner instead of ``<Your Nick>``.
```

Side note: Connecting to Libera with no `owner_account` set will make Sopel [nag the user about account matching](https://github.com/sopel-irc/sopel/blob/72493dcd4bd0b44d88a3a72f6c1948cfbd695ecd/sopel/coretasks.py#L399-L411) every time it starts, because Libera supports `account-tag`:

> This network supports using network services to identify you as my owner, rather than just matching your nickname. This is much more secure. If you'd like to do this, make sure you're logged in and reply with ".useserviceauth"

Reply:
Yes I know! That's how I usually set up my bot: I run it and let it ask me to use the command. 😁

Reply:
That, um, definitely starts users early on learning that Sopel will clobber comments in the config file 😅